src/components/SearchResultSorter.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import { Platform, Pressable, SafeAreaView, StyleSheet, Text, TextStyle, View, ViewStyle } from 'react-native'
+import BottomSheet from './BottomSheet'
+import Button from './Button'
+import SortOptions from './SortOptions'
+import {SortOptions as SortOptionsType} from '@typings/Search'
+import {sortOption as sortBy} from '../cache'
+
+const options: {label: string, value: SortOptionsType}[] = [
+  { label: 'Name: A to Z', value: 'name_asc' },
+  { label: 'Name: Z to A', value: 'name_desc' },
+  { label: 'Distance: Near to Far', value: 'distance_asc' },
+  { label: 'Distance: Far to Near', value: 'distance_desc' },
+  { label: 'Rating: High to Low', value: 'rating_asc' },
+  { label: 'Rating: Low to High', value: 'rating_desc' }
+]
+
+const INITIAL_OPTION_INDEX = 4
```

Comment:
Either move this to the top of the file and annotate it as a config variable, or make it an optional param for the component.